### PR TITLE
add goimports action

### PR DESCRIPTION
**Pull request by tomokinat**
This PR adds a github action which verifies the code format with goimports and also applies goimports itself to the code base (only one line diff).

### CODE REVIEW

**Comment by tomokinat** [comment]
### Examples
pass: https://github.com/tomokinat/webpackager/commit/5ee12877d2e0a77a04f7b91578b843c14052555b/checks?check_suite_id=391474983
fail: https://github.com/tomokinat/webpackager/commit/0c26dd86f970389f2b5f645d566076e8b0a79558/checks?check_suite_id=391471501

File: .github/workflows/goimports.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Validate code format with goimports
+on: [push, pull_request]
+jobs:
+  validate:
```
Comment (yuizumi) [inline .github/workflows/goimports.yml]: `check-format`?

Comment (tomokinat) [reply .github/workflows/goimports.yml]: Done.

File: .github/workflows/goimports.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Validate code format with goimports
```
[the remark this answers is not in the document]
Comment (tomokinat) [reply .github/workflows/goimports.yml]: Done.

File: .github/workflows/goimports.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v1
+
+    - name: Install goimports
+      run: go get golang.org/x/tools/cmd/goimports
+
+    - name: Check diff
```
[the remark this answers is not in the document]
Comment (tomokinat) [reply .github/workflows/goimports.yml]: Done.

**Comment by tomokinat** [comment]
FYI: the result of the commit 002a3c6 https://github.com/tomokinat/webpackager/commit/002a3c64d248ac7fc26a0c83dfce94b8e9b8eb64/checks?check_suite_id=391588644

**Pull request merged**